feature_creation_scripts/antarctic_ocean_regions/antarctic_ocean_regions.py

```diff
@@ -0,0 +1,280 @@
+             "coordinates": [[[lon0, lat0],
+                              [lon1, lat0],
+                              [lon1, lat1],
+                              [lon0, lat1],
+                              [lon0, lat0]]]}})
+    return fc
+
+
+def split_rectangle(lon0, lon1, lat0, lat1, name, author, tags, fcContour):
+    fc = make_rectangle(lon0, lon1, lat0, lat1, name, author, tags)
+
+    fcDeep = fc.difference(fcContour)
+
+    props = fcDeep.features[0]['properties']
+    props['name'] = props['name'] + ' Deep'
+    props['tags'] = props['tags'] + ';Deep'
+    props['zmin'] = -1000.
+    props['zmax'] = -400.
+
+    fcShelf = fc.difference(fcDeep)
+
+    props = fcShelf.features[0]['properties']
+    props['name'] = props['name'] + ' Shelf'
+    props['tags'] = props['tags'] + ';Shelf'
+    props['zmin'] = -1000.
+    props['zmax'] = -200.
+
+    fc.merge(fcDeep)
+    fc.merge(fcShelf)
+
+    return fc
+
+
+def main():
+    author = 'Xylar Asay-Davis'
+    timTags = 'Antarctic;Timmermann'
+    kusTags = 'Antarctic;Kusahara'
+
+    # make a geometric fieatures object that knows about the geometric data
+    # cache up a couple of directories
+    gf = GeometricFeatures('../../geometric_data')
+
+    bedmap2_bin_to_netcdf('bedmap2.nc')
+
+    fcContour700 = get_longest_contour(contourValue=-700., author=author)
+    fcContour800 = get_longest_contour(contourValue=-800., author=author)
+
+    fc = FeatureCollection()
+
+    fc.merge(split_rectangle(
+        lon0=-63., lon1=0., lat0=-80., lat1=-65., name='Weddell Sea',
+        author=author, tags=timTags, fcContour=fcContour800))
+
+    fc.merge(split_rectangle(
+        lon0=-30., lon1=45., lat0=-80., lat1=-65., name='Eastern Weddell Sea',
+        author=author, tags=kusTags, fcContour=fcContour800))
+
+    fc.merge(split_rectangle(
+        lon0=-63., lon1=-30., lat0=-80., lat1=-65., name='Western Weddell Sea',
+        author=author, tags=kusTags, fcContour=fcContour800))
+
+    fc.merge(split_rectangle(
+        lon0=-100., lon1=-63., lat0=-80., lat1=-67., name='Bellingshausen Sea',
+        author=author, tags=timTags, fcContour=fcContour700))
+
+    fc.merge(split_rectangle(
+        lon0=-140., lon1=-100., lat0=-80., lat1=-67., name='Amundsen Sea',
+        author=author, tags=timTags, fcContour=fcContour800))
+
+    fc.merge(split_rectangle(
+        lon0=-180., lon1=-140., lat0=-80., lat1=-67., name='Eastern Ross Sea',
+        author=author, tags=timTags, fcContour=fcContour700))
+
+    fc.merge(split_rectangle(
+        lon0=160., lon1=180., lat0=-80., lat1=-67., name='Western Ross Sea',
+        author=author, tags=timTags, fcContour=fcContour700))
+
+    fc.merge(split_rectangle(
+        lon0=45., lon1=160., lat0=-80., lat1=-62., name='East Antarctic Seas',
+        author=author, tags=kusTags, fcContour=fcContour800))
+
+    fc.merge(make_rectangle(
+        lon0=-180., lon1=180., lat0=-80., lat1=-60., name='Southern Ocean 60S',
+        author=author, tags=timTags))
```

Comment:
we probably don't need this region (SO 60S), right? it's probably here because that's what wikipedia says the SO is, or something like that, but we always agreed that this did not represent the SO.

Reply:
Again, this is the definition used by Timmermann and Hellmer.  I'd really like to keep it for comparison with them.

Reply:
I could add another plot with the "full" Southern Ocean (which we have extending to 48 S).

Reply:
no, no need to do that. I thought this was a legacy of the first SO region we had originally. I am surprised the paper uses this as SO definition though..

Reply:
Done.

Reply:
I went ahead and added zmin/zmax to the full Southern Ocean feature so we can use that too.